app/assets/stylesheets/application.scss

```diff
@@ -11,6 +11,37 @@
  * It is generally better to create a new file per style scope.
  *
  */
-
-@import "bootstrap-sprockets";
 @import "bootstrap";
+
+div.jumbotron {
+  padding: 0.5rem 2rem 0.5rem;
+  margin-bottom: 0.5rem;
+}
+
+.jumbotron a:hover{
+  text-decoration: none;
+}
+
+.desk-card{
+  margin: 0.5rem;
+}
+
+.desk-card-col{
+  max-width: 15em;
+  min-width: 15em;
+}
+
+.desk-row{
+  padding-left: 5em;
+  padding-right: 5em
+}
+
+.text-center > .alert{
+  padding: 0.3rem 1.25rem;
+  margin-bottom: 0.5rem;
+}
+
+body > .alert, body > .alert > .close
+{
+  padding: 0.3rem 1.25rem 0.3rem 1.25rem;
+}
```

Comment:
I'm not sure if this is the case, but if these classes are specific to the index page, they should be in a different css file. Application is generally reserved for universally applicable (and used) styles.

I was taught that the assets hierarchy basically mirrors that of views, in case you make new files.

Reply:
Good point Dom, that's smart